docs/v2.4/releaseNotes.yml

```diff
@@ -54,6 +54,30 @@ items:
         docs: reference/client
         image: telepresence-2.4.4-gather-logs.png
 
+      - type: feature
+        title: Pod CIDR strategy is configurable in Helm chart
+        body: >-
+          Telepresence now enables you to directly configure how to get
+          pod CIDRs when deploying Telepresence with the Helm chart.
+          The default behavior remains the same. We've also introduced
+          the ability to explicitly set what the pod CIDRs should be.
+        docs: install/helm
+
+      - type: bugfix
+        title: Compute pod CIDRs more efficiently
+        body: >-
+          When computing subnets using the pod CIDRs, the traffic-manager
+          now uses less CPU-cycles.
+        docs: reference/routing/#subnets
+
+      - type: bugfix
+        title: General CPU improvements in traffic-manager
```

Comment:
I think the body is okay, but i would change this title to something like `Prevent busy loop in traffic-manager`